src/Microsoft.PackageManagement.ArchiverProviders/project.json

```diff
@@ -44,8 +60,7 @@
         "netstandard1.6": {
             "imports": ["dnxcore50"],
             "buildOptions": {
-                "define": ["CORECLR"],
-                "debugType": "portable"
```

Comment:
Hm... I don't know why this was here. I was pretty sure it's the default, but it was purposefully added in https://github.com/PowerShell/PowerShell/pull/1959.

OH. [It defaults to `full` on Windows](https://github.com/dotnet/cli/blob/04f40f906dce2678d80fb9787e68de76ee6bf57e/src/dotnet/commands/dotnet-compile-csc/Program.cs#L201), and the previous PR overrode it.